src/flow_solver/flow_solver_factory.cpp

```diff
@@ -57,13 +58,23 @@ ::select_flow_case(const Parameters::AllParameters *const parameters_input,
             std::shared_ptr<FlowSolverCaseBase<dim, nstate>> flow_solver_case = std::make_shared<GaussianBump<dim, nstate>>(parameters_input);
             return std::make_unique<FlowSolver<dim, nstate>>(parameters_input, flow_solver_case, parameter_handler_input);
         }
+    } else if (flow_type == FlowCaseEnum::isentropic_vortex){
+        if constexpr (nstate==dim+2 && dim!=1){
+            std::shared_ptr<FlowSolverCaseBase<dim, nstate>> flow_solver_case = std::make_shared<PeriodicEntropyTests<dim,nstate>>(parameters_input);
+            return std::make_unique<FlowSolver<dim,nstate>>(parameters_input, flow_solver_case, parameter_handler_input);
+        }
+    } else if (flow_type == FlowCaseEnum::kelvin_helmholtz_instability){
+        if constexpr (dim>1 && nstate==dim+2){
+            std::shared_ptr<FlowSolverCaseBase<dim, nstate>> flow_solver_case = std::make_shared<PeriodicTurbulence<dim,nstate>>(parameters_input);
```

Comment:
It will use PeriodicEntropyTests in the future. Until just today, I was using the PeriodicTurbulence case temporarily (which is why the results I have sent to you so far don't have have entropy saved!). Both of them use straight periodic grids, so the physics is identical. I prefer to have the two Euler tests in a separate flow case as PeriodicTurbulence is getting quite complex already. Now as I'm cleaning my branches, I'm migrating the two to use the same flowsolver case. 

Thanks for pointing it out!